Source/Core/Core/WiiUtils.cpp

```diff
@@ -160,6 +160,18 @@ bool InstallWAD(const std::string& wad_path)
   return InstallWAD(ios, DiscIO::WiiWAD{wad_path}, InstallType::Permanent);
 }
 
+bool UninstallTitle(u64 title_id)
+{
+  IOS::HLE::Kernel ios;
+  return ios.GetES()->DeleteTitleContent(title_id) == IOS::HLE::IPC_SUCCESS;
+}
+
+bool IsTitleInstalled(u64 title_id)
```

Comment:
To check if a title is installed, just checking if a TMD is present is not enough because removing titles (from the Wii Menu for example) removes all private title content files and leaves the TMD on the NAND.

I suggest calling `ES::GetStoredContentsFromTMD` and comparing the stored content list against the TMD content list.

Reply:
You're right. I think I somehow misread the old logic as checking if there is a TMD rather than checking if there's something that isn't a TMD...